### PR TITLE
File ICLA for Manuel Recena

### PR DESCRIPTION
**Pull request by recena**
I'd like to be part (offically) of the Jenkins project.

### CODE REVIEW

File: collected/icla/recena/committer.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name=Manuel RecenaA
```
Comment (oleg-nenashev) [inline collected/icla/recena/committer.properties]: RecenaA?

Comment (recena) [reply collected/icla/recena/committer.properties]: Ups

**Comment by recena** [comment]
@rtyler Should this request be reviewed or approved by the Governance Board or similar?

**Comment by oleg-nenashev** [comment]
@recena What do you exactly want to do with this SLA? Be a part of the Core team with Write access to the core/modules? Or what?

**Comment by recena** [comment]
@oleg-nenashev I'm trying to understand [these steps](https://github.com/jenkinsci/infra-cla#how-to-accept-pr).

**Pull request merged**